tests/integration-tests/arweave-and-3box/contracts/Contract.sol

```diff
@@ -0,0 +1,21 @@
+pragma solidity ^0.6.1;
+
+contract Contract {
+  event Trigger();
+
+  constructor() public {
+    emit Trigger();
+  }
+
+  function exampleFunction(string memory) public pure returns (string memory) {
+    return "string -> string";
+  }
+
+  function exampleFunction(uint256) public pure returns (string memory) {
+    return "uint256 -> string";
+  }
+
+  function exampleFunction(bytes32) public pure returns (uint256) {
+    return 256;
+  }
```

Comment:
It could be misleading that we have these here. Shall we drop them from this test? The contract doesn't have to do anything but emit a trigger.

Reply:
True, done.